personalize.api.php

```diff
@@ -319,24 +319,12 @@ function hook_personalize_delete_link($option_set) {
 }
 
 /**
- * Returns a render array or themed output for a report on the specified agent.
+ * Allows alteration of the wizard steps.
```

Comment:
Sorry.... I would include that this are the wizard steps for managing a campaign.... </nitpick>